### PR TITLE
[WIP] Hooks

### PR DESCRIPTION
**Pull request by Kaik**
In progress #300 and #299

### CODE REVIEW

**Comment by craigh** [comment]
probably need to rebase. the old style config needed correction

**Comment by Kaik** [comment]
Ok :+1:

**Comment by Kaik** [comment]
This is not finished, just partly refactored and cleaned.
Will come back to this as soon as I will finish with module main functionality.

File: HookHander/AbstractHookHandler.php [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+ * This work is contributed to the Zikula Foundation under one or more
+ * Contributor Agreements and licensed to You under the following license:
+ *
+ * @license MIT
+ *
+ * Please see the NOTICE file distributed with this source code for further
+ * information regarding copyright and licensing.
+ */
+
+namespace Zikula\DizkusModule;
```
Comment (craigh) [inline HookHander/AbstractHookHandler.php]: this directory has a typo `Hander` vs `Handler` and also the namespace should include the parent directory e.g. `namespace Zikula\DizkusModule\HookHandler`

Comment (Kaik) [inline HookHander/AbstractHookHandler.php]: Fixed in awaiting PR as well as wrong namespace.

**Pull request merged**